cypress/fixtures/profile.json

```diff
@@ -0,0 +1,5 @@
+{
+  "id": 8739,
+  "name": "Jane",
+  "email": "jane@example.com"
+}
```

Comment:
What is this fixture used for?

Reply:
We forgot to delete this when we were deleting extra files that cypress creates.